week-5/C-promises/exercise.js

```diff
@@ -11,8 +11,15 @@
   Insert the result of promise1 into the #exercise1 element. Hint: .then()
 */
 function exercise1() {
-  var promise1 = resolvedPromise()
-}
+  var promise1 = resolvedPromise();
+  promise1.then(function (){
+    var promise = document.querySelector('#exercise1');
+    promise.innerText = "promise1";
```

Comment:
I don't think this is quite right. `promise1` will resolve back with a result that is a string. You need to take that string and insert it into the DOM. Here you are making your own string and inserting that into the DOM.

Hint: if you add a parameter to the `.then` handler function can then `console.log` it, that is what we need to insert into the DOM:

```js
function exercise1() {
  var promise1 = resolvedPromise();
  promise1.then(function(result) {
    console.log(result)
    // Then we need to insert the result variable into the DOM
  })
}
```

Does that make sense?